iris/models/global_search_intent/train.py

```
@@ -0,0 +1,225 @@
+print("=== Installing dependencies ===")
+pkgs = [
+    "setfit",
+    "onnxruntime",
+    "optimum[onnxruntime]",
+    "pandas",
+    "scikit-learn",
+    "datasets",
+    "joblib",
+]
+for pkg in pkgs:
+    subprocess.check_call(  # nosec B603
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)